t04_refactored.py

```diff
@@ -166,7 +166,107 @@ def team_11_adv():
 
 def team_12_adv():
     pass
-    # TODO Add your code here
+    # TEAM 12
+    """"
+      
+    https://docs.google.com/document/d/1EMslJjPk-r_QKwfR1_Ipu-zlEJcsSvobid9GAsLtMaE/edit?usp=sharing
+    
+    """
+
+    hasSword = True
+    goblindead = True
+    ######Intro
+    print("You stand in your makeshift campsite, a pitiful fire smoulders, sending black smoke to the sky.")
+    sleep(delay)
+    print("To the North: The Mystic Wood lies, fabeled to hold a great treasure.")
+    sleep(delay)
+    print("To the South: The Mouth of a cave, you can hear noises creep from within.")
+    sleep(delay)
+    print("To the West: A long road that you came from.")
+    sleep(delay)
+    print("To the East: A vast meadow of yellow flowers.")
+    sleep(delay)
+    ############
+    direction = input("Choose a direction or action[Search the camp/North/South/East/West]: ")
+
+    # CHECK FOR VALID INPUT #
+
+    while direction != "north" and direction != "south" and direction != "east" and direction != "west" and direction != "search camp":
```

Comment:
All of these cannot be true at the same time. Did you mean `or` instead of `and`?